ios/PresentationLayer/UIToolkit/Plugins/SwiftGenPlugin/Plugin.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Created by Petr Chmelar on 08.09.2022
+//  Copyright © 2022 Matee. All rights reserved.
+//
+
+import PackagePlugin
+
+@main
+struct SwiftGenPlugins: BuildToolPlugin {
+    func createBuildCommands(context: PluginContext, target: Target) async throws -> [Command] {
+        let configurations: [Path] = [context.package.directory, target.directory]
+            .map { $0.appending("swiftgen.yml") }
+        
```

Comment:
Maybe you need to filter configuration locations

.filter { FileManager.default.fileExists(atPath: $0.string) }